Pod/Classes/Acknowledgements/AcknowledgementViewController.swift

```diff
@@ -83,3 +91,30 @@ private extension AcknowledgementViewController {
         labelView.attributedText = licenseFormatter(viewModel.license)
     }
 }
+
+private class VerticalScrollView: UIScrollView {
+
+    private var subviewWidthConstraints: [NSLayoutConstraint] = []
```

Comment:
#? shouldn't this be modified by adding the new constraints to them in `updateContentInset`? Otherwise, it's always empty. Alternatively, remove this.